src/frontend/ts/state/preferenceState.ts

```diff
@@ -0,0 +1,8 @@
+interface PreferenceState {
+  pieceTheme: string | null;
+  boardTheme: string | null;
+  setPieceTheme: (theme: string) => void;
+  setBoardTheme: (theme: string) => void;
```

Comment:
Optional, but if we want to be safer, we could define the piece and board themes as enumerations (see `types/urbitChess`) and then require that the `PreferenceState` take a `BoardTheme`/`PieceTheme` enum. This would give us stricter typing, which is safer.

I think it's the right thing to do, but it doesn't have to go into this PR: we can just add it as an issue for now, and finish up this PR.